packages/core-utils/src/watcher.ts

```diff
@@ -58,16 +58,39 @@ export class Watcher {
     }
 
     const msgHashes = []
+    const sentMessageEventId = ethers.utils.id('SentMessage(address,address,bytes,uint256,uint256)')
+    const l2CrossDomainMessengerRelayAbi = [
+      'function relayMessage(address _target,address _sender,bytes memory _message,uint256 _messageNonce)',
+    ]
+    const l2CrossDomainMessengerRelayinterface = new ethers.utils.Interface(
+      l2CrossDomainMessengerRelayAbi
+    )
     for (const log of receipt.logs) {
       if (
         log.address === layer.messengerAddress &&
-        log.topics[0] === ethers.utils.id('SentMessage(bytes)')
+        log.topics[0] ===
```

Comment:
Seems like this file needs to be linted but otherwise looks good to me.

Reply:
Thanks! Fixed and rebased